### PR TITLE
Add updates to gcp doc and terraform

### PR DESCRIPTION
**Pull request by strongjz**
Cleaning up the walk through after some folks tested it out.

### CODE REVIEW

File: terraform/google_ca.tf [hunk 1]
```diff
@@ -14,9 +14,6 @@ resource "google_privateca_ca_pool" "default" {
       allow_config_based_issuance = true
     }
   }
-  lifecycle {
```
Comment (vaikas) [inline terraform/google_ca.tf:17]: Is this removal intentional? If it is, this might warrant a comment stating why we do not prevent accidental deletions.

**Review by priyawadhwa — APPROVED** [review]

File: terraform/tekton.tf [hunk 2]
```diff
@@ -76,8 +76,11 @@ data "kubernetes_secret" "ctlog-public-key" {
   }
 }
 
+resource "time_sleep" "wait_30_seconds" {
+  create_duration = "30s"
+}
 resource "kubernetes_secret" "ctlog-public-key" {
-  depends_on = [helm_release.sigstore_scaffold]
+  depends_on = [helm_release.sigstore_scaffold, time_sleep.wait_30_seconds] #ctlog times a minute to come up
```
Comment (vaikas) [inline terraform/tekton.tf:83]: Wonder if we should have (in the future, doesn't have to be part of this PR), a job that waits and checks that things are up instead of relying on the sleep.

**Review by vaikas — COMMENTED** [review]
Thanks for cleaning this up, couple of small questions / thoughts.

File: terraform/google_ca.tf (hunk 1 above)
Comment (strongjz) [reply terraform/google_ca.tf:17]: added it back, probably from testing it was removed.
